<launch>
    <node pkg="position_control" type="laser_rc_control" name="laser_rc_control" output="screen">
    <!-- node parameters -->
    <param name="alt_P1" type="double" value= "1.3" />
    <param name="vz_P" type="double" value=  "0.5"/>
    <param name="vz_I" type="double" value=  "0.05"/>
    <param name="vz_D" type="double" value=  "0.2"/>
  
    <param name="xy_P" type="double" value=  "0.5"/>
    <param name="vxy_P" type="double" value= "0.3"/>
    <param name="vxy_I" type="double" value= "0.6"/>
    <param name="vxy_D" type="double" value= "0.25"/>
    <param name="tag_X" type="double" value= "10"/>
    <param name="tag_Y" type="double" value= "10"/>
    <param name="tag_Z" type="double" value= "3"/>
    <param name="max_V" type="double" value= "0.5"/>
    <param name="max_ACC" type="double" value= "1.0"/>

    <param name="max_pitch_roll" type="double" value= "35.0"/>
    </node>
    
</launch>
